### DISCUSSION TITLE
Table not rendered correctly

### DISCUSSION THREAD

**Discussion by muhirza** [Power-Components/livewire-powergrid#1021] [BUG]
I just installed and follow everything that shows on the documentation
and the table not rendered correctly:
![Screenshot (179)](https://github.com/Power-Components/livewire-powergrid/assets/65381663/02c2756f-b6f7-4db2-ad74-6dbb6d7657e2)

![Web capture_8-6-2023_114545_localhost](https://github.com/Power-Components/livewire-powergrid/assets/65381663/e6d305a7-f423-4db8-b504-e37e529555f7)

first picture showing that I used filter inline, second I use filter outside.

I already installed livewire based on the instructionn at their web, and also I already installed tailwind css using this link below.
https://tailwindcss.com/docs/guides/laravel

I just feel the between the tailwind css or alpine js does not work correctly yet i still cannot found out the issue. I try the bootstrap one and it shows correctly, yet still i need to build from tailwind css for this projects

thank you

**Comment by luanfreitasdev** [comment]
Have a look here, maybe this will help you.

https://github.com/Power-Components/powergrid-demo-misc
